src/__test__/App.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+/* eslint-disable */
+import store from '../redux/store';
+import App from '../App';
+/* eslint-enable */
```

Comment:
- [ ] kindly make sure you fix the following linter error instead of disabling it for best practices. This is because of a bad path with a typing error. Below is how you can fix it:

```JavaScript
// Bad
import store from '../redux/store';

// Good
import store from '../Redux/store';
```